### ISSUE TITLE
Add a “next problem” button to the bottom of the screen

### ISSUE THREAD

**Issue by benetechMatt** [benetech/MathShare#970]

**Comment by benetechMatt** [comment]
![mockup of the problem solving screen. at the bottom, below the math editor tools palette there are three buttons. In the middle is "finish" which currently exists.  On the left is a button with a left arrow and the text "previous problem". On the right is a button with a right arrow and the text "next problem". The button is on the far right side of the screen in the same row as the other two buttons](https://user-images.githubusercontent.com/49413678/75600009-041bf700-5a70-11ea-87b8-fddb6eb51815.png)

Add padding at the bottom so that the Intercom button can be below the "next problem button"

@alexrcabral how should the buttons be styled so they don't draw the users attention too much compared to finish.  I'm assuming students will use "finish" more often that the next button, agree?

**Comment by alexrcabral** [comment]
Depends on the student. If they like more steady, consecutive work and want to finish problems before moving on to the next, they might use the finish button more. But if they like to jump around between problems, connect patterns, or complete the same step of each problem, they're going to use next problem more frequently.

I think the way it is laid out in the mockup is fine.

**Comment by johnhbenetech** [comment]
I assume the previous and next buttons should be disabled for the first and last problems respectively?

or do you want it to cycle?

**Comment by alexrcabral** [comment]
(Copied from #969): The "next" and "previous" buttons should be visibly inactive (eg greyed out) and not able to be interacted with for non-applicable pages. First problem should not have an active "previous" button, last problem should not have an active "next" button.

**Comment by jscholes** [comment]
#### Intercom Positioning

@benetechMatt already noted the potential for issues with the Intercom component positioning.  Namely, controls could end up being overlapped and hidden.  But just adding padding won't help, because the Intercom Messenger Component is rendered with `position: fixed;`.  This really needs to be addressed so that it follows better responsive practice.

There is also an issue with the Intercom button being the very last control on the page for a screen reader user, which significantly harms its discoverability.  Maybe these could be addressed together, to move it into the header and fix its positioning.

**Comment by benetechMatt** [comment]
We'll remove the intercom button for the problem solving page for all users #1063

**Comment by johnhbenetech** [comment]
As discussed on call:
* First problem will not render 'Previous Button'
* Last problem will not render 'Next Button'

**Comment by jscholes** [comment]
@johnhbenetech The buttons won't be rendered at all?  Or they'll be present but disabled.

**Comment by johnhbenetech** [comment]
@jscholes I believe @alexrcabral asked them to not be rendered at all. Alex correct me if I'm wrong

**Comment by jscholes** [comment]
@johnhbenetech Thanks for clarifying.  See https://github.com/benetech/MathShare/issues/969#issuecomment-604152105.

**Comment by johnhbenetech** [comment]
We will go with disabled button approach due to lack of communication from product team

**Comment by abaranowska1** [comment]
"Next Problem" button was added to the bottom of the screen and  "wrap around" rotating button approach was used
![image](https://user-images.githubusercontent.com/61964176/82318281-df308380-99cf-11ea-93d4-afad4b8d2a23.png)
